coriolis_openstack_utils/cli/shell.py

```diff
@@ -50,15 +50,13 @@ def build_option_parser(self, description, version, argparse_kwargs=None):
     def run(self, argv):
         # display usage if no args provided:
         known_args, remainder = self.parser.parse_known_args()
-        if known_args.conf_file is None:
-            self.stderr.write(self.parser.format_usage())
-            return 1
-        CONF(
-            # NOTE: passing the whole of sys.argv[1:] will make
-            # oslo_conf error out with urecognized arguments:
-            ["--config-file", known_args.conf_file],
-            project=constants.PROJECT_NAME,
-            version=constants.PROJECT_VERSION)
+        if known_args.conf_file is not None:
+            CONF(
+                # NOTE: passing the whole of sys.argv[1:] will make
+                # oslo_conf error out with urecognized arguments:
+                ["--config-file", known_args.conf_file],
+                project=constants.PROJECT_NAME,
+                version=constants.PROJECT_VERSION)
```

Comment:
Shouldn't add an `else` clause where we error out if `known_args.conf_file is None`?

Reply:
if known_args.conf_file is none, in the case `coriolis-openstack-util help migrate user` for example, we shouldn't error out.